src/sentry/static/sentry/app/stores/hookStore.jsx

```diff
@@ -36,6 +33,14 @@ const validHookNames = new Set([
   'sidebar:bottom-items',
   'sidebar:item-label',
 
+  // Onboarding experience
+  // TODO(epurkhiser): These all should become less getsentry specific
```

Comment:
No but we try to keep getsentry wording outside of the sentry open source project. So ideally instead of these being specific hooks *for getsentry* they are more general.

This comment could probably be a little better though since it's primarily `onboarding-survey-url` that is highly specific to the getsentry feature, and should probably be something like `additional-onboarding-steps`.